### ISSUE TITLE
Offline editor download page: top paragraph not centered

### ISSUE THREAD

**Issue by cwillisf** [LLK/scratch-www#1381]
When the download page is displayed in a wide window, the paragraph at the top of the page ("You can install the Scratch 2.0 editor....") is aligned left and wrapped to an unnecessarily narrow column width.

Note that this problem existed on the old page, but IMO stands out more on the new version of the page.

Screenshot:
![image](https://user-images.githubusercontent.com/7019101/27609210-04c9c48a-5b3f-11e7-8f4e-13fbe4c6f57b.png)

**Comment by TheGrits** [comment]
@cwillisf 

This was purposefully done to match the design of other newly migrated pages, such as the [Educators](https://scratch.mit.edu/educators/) and [Developers](https://scratch.mit.edu/developers) pages.

Another option could be to implement the plainer design used by the info pages, such as the [FAQ page](https://scratch.mit.edu/info/faq).

**Comment by jwzimmer** [comment]
The Educators & Download pages seem to behave differently at smaller widths, and there seems to be padding to the left of the Scratch logo in the navbar that isn't present to the left of the text at the top of the Download page.

<img width="590" alt="screen shot 2017-06-29 at 10 17 35 am" src="https://user-images.githubusercontent.com/1320456/27692302-5a1e4c0a-5cb4-11e7-870a-0deb3a1fc12a.png">
<img width="934" alt="screen shot 2017-06-29 at 10 17 13 am" src="https://user-images.githubusercontent.com/1320456/27692300-5a16ae00-5cb4-11e7-9485-239bf4ac2c0d.png">
<img width="935" alt="screen shot 2017-06-29 at 10 16 30 am" src="https://user-images.githubusercontent.com/1320456/27692299-5a168e5c-5cb4-11e7-8f26-3976af168d4d.png">
<img width="402" alt="screen shot 2017-06-29 at 10 15 09 am" src="https://user-images.githubusercontent.com/1320456/27692301-5a199a70-5cb4-11e7-9c3d-46ebb90dcc97.png">

**Comment by TheGrits** [comment]
@jwzimmer Thank you, I understand the issue much more clearly now!

I don't have time to fix it at present, but may try to fix it at a later point.

Hey @St19Galla maybe you would be interested in tackling this `help-wanted` issue?

**Comment by mewtaylor** [comment]
@jwzimmer I feel like I'm seeing that padding misalignment on both pages, including Download – it looks like the logo is not being spaced out all the way to the edge of the box in all cases on that size of a screen.

As for the text on the download page taking up 100% – @carljbowman I think in our typography/global stylings we have text take up 100% on mobile. Should we consider standardizing a margin at that screen size, given that this is not the first time we're looking into this? Or do we just want to treat this as another exception to that rule?

**Comment by GreenBayRules** [comment]
I'll try this

**Comment by GreenBayRules** [comment]
Ok, created a pull request above ^

**Comment by thisandagain** [comment]
Agreed with @mewtaylor that we probably need to normalize / standardize the rules here regarding when to center and assigning a standard margin for mobile (rather than the awkward spacing we have now). Until this is decided I'm going to remove "help-wanted".

**Comment by DeleteThisAcount** [comment]
offtopic question, @cwillisf why is the navbar orange?

**Comment by VutonDesign** [comment]
I think it’s because it’s a testing/development version they’re using.

**Comment by chrisgarrity** [comment]
@DeleteThisAcount eThisAcount as @VutonDesign said - our testing environment has an orange navbar so we can easily see if we're on the staging server or the live site.
